### PR TITLE
User activity logs

### LINKED ISSUE #693: User activity logs
Create a page with all user activites with filter for daterange and users

### PR DESCRIPTION
**Pull request by tsumo**
## PR includes

-   [x] Feature

## Related issues

Resolve #693

### CODE REVIEW

File: src/components/UserListItem/UserListItem.tsx [hunk 1]
```diff
@@ -11,13 +10,13 @@ import { UserPic } from '../UserPic';
 import { tr } from './UserListItem.i18n';
 
 interface UserListItemProps {
-    user: User;
+    user: { id: string; name: string | null; email: string; active: boolean };
```
Comment (DenisVorop) [inline src/components/UserListItem/UserListItem.tsx:13]: maybe would be better?
```ts 
ComponentProps<typeof UserPic>['user']
```

File: src/components/UserActivityPage/UserActivityPage.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+export const UserActivityPage = ({ userId }: UserActivityPageProps) => {
+    const activityQuery = trpc.historyEvent.getUserActivity.useInfiniteQuery(
+        { userId },
+        { getNextPageParam: (lastPage) => lastPage.nextCursor },
+    );
+    const events = activityQuery.data?.pages.flatMap((page) => page.events);
+    const loadMore = activityQuery.fetchNextPage;
+    const hasNext = activityQuery.hasNextPage;
+    const userQuery = trpc.user.getById.useQuery(userId);
```
Comment (DenisVorop) [inline src/components/UserActivityPage/UserActivityPage.tsx:57]: why are you dont use destructuring? 

```ts
const { data: user } = trpc.
```

File: src/components/UserActivityPage/UserActivityPage.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    events,
+    loadMore,
+    hasNext,
+}: {
+    user: TrpcRouterOutput['user']['getById'];
+    events: TrpcRouterOutput['historyEvent']['getUserActivity']['events'];
+    loadMore: VoidFunction;
+    hasNext?: boolean;
+}) => {
+    const actingUser = useMemo(() => ({ id: user.id, name: user.name, email: user.email }), [user]);
```
Comment (DenisVorop) [inline src/components/UserActivityPage/UserActivityPage.tsx:28]: for what?

why not `actingUser: user` on line 38 ?

File: src/components/UserListItem/UserListItem.tsx (hunk 1 above)
Comment (tsumo) [reply src/components/UserListItem/UserListItem.tsx:13]: Doesn't have id

File: src/components/UserActivityPage/UserActivityPage.tsx (hunk 2 above)
Comment (tsumo) [reply src/components/UserActivityPage/UserActivityPage.tsx]: fixed

File: src/components/UserActivityPage/UserActivityPage.tsx (hunk 3 above)
Comment (tsumo) [reply src/components/UserActivityPage/UserActivityPage.tsx:28]: Excess data was very confusing during development and debugging

Comment (tsumo) [reply src/components/UserActivityPage/UserActivityPage.tsx]: But you are right, i fixed it

**Pull request merged**